databricks/koalas/tests/test_dataframe.py

```diff
@@ -307,6 +307,26 @@ def test_sort_values(self):
         self.assert_eq(kdf.sort_values('b', inplace=True), pdf.sort_values('b', inplace=True))
         self.assert_eq(repr(kdf), repr(pdf))
 
+    def test_sort_index(self):
+        pdf = pd.DataFrame({'A': [2, 1, np.nan]}, index=['b', 'a', np.nan])
```

Comment:
I suggest that the DataFrame contains two columns.

Reply:
Changed with 0db87fd.